examples/apigee/v1beta1/syncauthorization.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apigee.gcp.upbound.io/v1beta2
+kind: Organization
```

Comment:
Could you please put the `SyncAuthorization` example at the beginning of the YAML file?